### PR TITLE
Feat/swarinfo 264 integrate usb driver

### PR DESCRIPTION
**Pull request by cquesnel**
All C code is generated. Do not review!

### CODE REVIEW

File: cmake/usb/FindUSB.cmake [hunk 1]
```diff
@@ -0,0 +1,115 @@
+if(NOT USB_FIND_COMPONENTS)
```
Comment (xgroleau) [inline cmake/usb/FindUSB.cmake:1]: Should the whole file go in stm32/FindUSB.cmake ,like the FindBSP, FindHAL, FindCMSIS? Or if we have another chip we will have both port in the same file?

File: src/Main.cpp [hunk 2]
```diff
@@ -219,7 +219,7 @@ int main(int argc, char** argv) {
     s_tcpDispatchTask.start();
     s_uartMessageSender.start();
     s_tcpMessageSender.start();
-    s_spiMessageSender.start();
+    // s_spiMessageSender.start();
```
Comment (xgroleau) [inline src/Main.cpp:222]: Forgot to uncomment

**Review by xgroleau — APPROVED** [review]
LGTM 👍

File: src/Main.cpp (hunk 2 above)
Comment (cquesnel) [reply src/Main.cpp:222]: No it breaks if we don't have an ESP plugged in. So until we all work with a HiveSight, we will leave commented

File: cmake/usb/FindUSB.cmake (hunk 1 above)
Comment (cquesnel) [reply cmake/usb/FindUSB.cmake:1]: It could go there!

**Pull request merged**